server/service/oauth.go

```diff
@@ -88,12 +98,83 @@ func (o *OAuth) GetUser(ctx context.Context, code string) (*model.User, error) {
 	var user githubUser
 	err = json.NewDecoder(resp.Body).Decode(&user)
 	if err != nil {
-		return nil, fmt.Errorf("can't parse github response: %s", err)
+		return nil, fmt.Errorf("can't parse github user response: %s", err)
+	}
+
+	if o.restrictAccess != "" {
+		if err := o.checkAccess(client, o.restrictAccess, user.Login); err != nil {
+			return nil, err
+		}
 	}
+
+	role := model.Requester
```

Comment:
We are using the `reviewer` and `requester` words for the same concept.
Maybe this PR is a good place to fix this.

Reply:
Wow, that's a good catch: we decide to call that role "requester", but this PR introduces a new concept colliding with an existing one.
Let's do `s/reviewer/requester/g`